docs/src/main/sphinx/connector/atop.rst

```diff
@@ -0,0 +1,66 @@
+==============
+Atop connector
+==============
+
+Overview
+--------
+
+The Atop connector supports reading disk utilization statistics from the ``atop``
+(Advanced System & Process Monitor) Linux server performance analysis tool.
+
+In order to use this connector, the machine on which the Trino worker is running
+needs to have the ``atop`` tool installed locally.
+
+Connector configuration
+-----------------------
+
+Create ``etc/catalog/atop.properties``
```

Comment:
do not use name of connector as name of catalog .. maybe use something mydisks or so

Reply:
Every other connector is using the name of the db from which is reading the data.
Can you detail a bit the reasoning behind this request?

Reply:
Typically the connector name details the technology used to store some data. But when querying the data with SQL you dont want to care about how it is stored.. you want to know what is is about. Therefore the catalog name should NOT indicate the RDBMS or whatever but instead be a short name for the data. In this case the data is about disk utilization on the workers .. so find a name that reflects that.

For most cases also you can use one connector for many catalogs (e.g. five different databases all on postgresql) .. that shows clearly how useless it is to use the connector name for the catalog file..

Reply:
I changed the name of the connector to `system_monitor`